sdcm/scylla_bench_thread.py

```diff
@@ -56,6 +69,34 @@ def run(self):
                         event.add_info(node=self.node, line=line, line_number=line_number).publish()
 
 
+class ScyllaBenchTimeSeriesWriteTimestampStartSearcher(FileFollowerThread):
```

Comment:
This is overkill, just provide `time.time()` to `-start-timestamp` on `write`, remember it and pick it up on read

Reply:
Will it be custom timestamp for cell when it is written or it is the value for twcs cluster key?
@roydahan what do you think?

Reply:
@dkropachev implement as you suggest.
running job to validate

Reply:
@dkropachev are you sure?
If so, @aleksbykov I think this is already the behaviour.

Reply:
I prefer to keep it like this if it works well

Reply:
Job is running succesfull with custom time. https://jenkins.scylladb.com/job/scylla-staging/job/abykov/job/longevity-twcs-test/2/consoleFull
According scylla-bench code, the starttime will be taken from -starttime if it tpassed,  if not it just take Time.Now(). so passing the timestamp to write and read mode doesn't require this overhead.

Reply:
@roydahan monitor is http://54.154.21.81:3000